### PR TITLE
chore(ui): update svelte toolchain

### PR DESCRIPTION
**Pull request by geigerzaehler**
We update all svelte related packages. This requires us to fix some typechecking errors.

### CODE REVIEW

File: ui/src/session.ts [hunk 1]
```diff
@@ -191,7 +191,7 @@ export const addSeed = async (seed: string): Promise<boolean> => {
   // running async remote validations in in the background. If we remove the
   // async then the seed input form will have to be submitted twice to take any
   // effect.
-  await seedValidation.validate(seed);
+  seedValidation.validate(seed);
```
Comment (xla) [inline ui/src/session.ts:194]: Is this safe, given the comment above?

File: rollup.app.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default {
       // we'll extract any component CSS out into
       // a separate file — better for performance
       css: css => {
-        css.write("public/bundle.css");
+        css.write("bundle.css");
```
Comment (xla) [inline rollup.app.js:26]: Why do we write to a different location?

File: ui/src/session.ts (hunk 1 above)
Comment (geigerzaehler) [reply ui/src/session.ts]: Removing this wasn’t fully intentional. I’ve reverted it in https://github.com/radicle-dev/radicle-upstream/pull/1215/commits/2c8bc6d0b24a6a829e85863b12ddc7f7f908dc94

File: rollup.app.js (hunk 2 above)
Comment (geigerzaehler) [reply rollup.app.js:26]: This was a [major breaking change](https://github.com/sveltejs/rollup-plugin-svelte/blob/master/CHANGELOG.md#600) in `svelte-rollup`. Now the path is relative to the output of the bundle.

**Review by xla — APPROVED** [review]
💶 ™️ 🐿 📇

**Review by rudolfs — DISMISSED** [review]

**Review by xla — APPROVED** [review]
♎️ 🙎 🏎 🌓

**Review by MeBrei — APPROVED** [review]

**Review by rudolfs — APPROVED** [review]

**Pull request merged**